[2025-03-19T19:05:25.333+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T19:05:25.628+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.get_name scheduled__2024-09-27T00:00:00+00:00 [queued]>
[2025-03-19T19:05:25.851+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.get_name scheduled__2024-09-27T00:00:00+00:00 [queued]>
[2025-03-19T19:05:25.854+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-19T19:05:25.952+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2024-09-27 00:00:00+00:00
[2025-03-19T19:05:26.078+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=9382) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T19:05:26.097+0000] {standard_task_runner.py:72} INFO - Started process 9396 to run task
[2025-03-19T19:05:26.075+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_Api_v2', 'get_name', 'scheduled__2024-09-27T00:00:00+00:00', '--job-id', '2722', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmptfaayf0e']
[2025-03-19T19:05:26.114+0000] {standard_task_runner.py:105} INFO - Job 2722: Subtask get_name
[2025-03-19T19:05:26.393+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_Api_v2.get_name scheduled__2024-09-27T00:00:00+00:00 [running]> on host 63fd4b143e0a
[2025-03-19T19:05:27.570+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_Api_v2' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2024-09-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-27T00:00:00+00:00'
[2025-03-19T19:05:27.577+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T19:05:27.598+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'John', 'last_name': 'Doe'}
[2025-03-19T19:05:27.685+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T19:05:27.687+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_Api_v2, task_id=get_name, run_id=scheduled__2024-09-27T00:00:00+00:00, execution_date=20240927T000000, start_date=20250319T190525, end_date=20250319T190527
[2025-03-19T19:05:27.828+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T19:05:28.059+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-19T19:05:28.063+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
